atcodertools/common/language.py

```diff
@@ -79,7 +79,7 @@ def from_name(cls, name: str):
     name="python",
     display_name="Python3",
     extension="py",
-    submission_lang_pattern=re.compile(".*Python3.*"),
+    submission_lang_pattern=re.compile(".*Python.*"),
```

Comment:
ここは既に他マージリクエストで対応したので衝突避けのために変更なしにします。
```suggestion
    submission_lang_pattern=re.compile(".*Python3.*"),
```